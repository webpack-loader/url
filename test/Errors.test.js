const loader = require('../src/index.js')

describe('Errors', () => {
  test('Validation Error', () => {
    const err = () => loader.call({
      query: { publicPath: 1 },
      emitFile: true
    })

    expect(err).toThrow()
    expect(err).toThrowErrorMatchingSnapshot()
  })
})
