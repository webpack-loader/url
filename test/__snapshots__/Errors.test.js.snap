// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors Validation Error 1`] = `
"URL Loader Invalid Options

options.publicPath should be {String}/{Function} (https://github.com/webpack-loader/url#publicpath)
"
`;
